mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala

```diff
@@ -123,21 +95,74 @@ class StopWordsRemover(override val uid: String)
   /** @group getParam */
   def getCaseSensitive: Boolean = $(caseSensitive)
 
-  setDefault(stopWords -> StopWords.English, caseSensitive -> false)
+  /**
+    * the language of stop words
+    * Default: "english"
+    * @group param
+    */
+  val language: Param[String] = new Param[String](this, "language", "stopwords language")
+
+  /** @group setParam */
+  def setLanguage(value: String): this.type = {
+    val lang = value.toLowerCase
+    require(StopWords.languageMap.contains(lang), s"$lang is not in language list")
+    set(language, lang)
+    set(stopWords, StopWords.languageMap(lang))
```

Comment:
Ditto. Do not set the stop words here. Implement the logic in `transform` instead.